### DISCUSSION TITLE
no DEBUG log

### DISCUSSION THREAD

**Discussion by gulaschnascher4000** [open-webui/open-webui#3803] [General]
# Bug Report

## Description
even if I adjust "DEBUG" in all sitatuions, I wont get any proper output.

**Bug Summary:**
INFO:config:GLOBAL_LOG_LEVEL: DEBUG
INFO:config:AUDIO_LOG_LEVEL: DEBUG
INFO:config:COMFYUI_LOG_LEVEL: DEBUG
INFO:config:CONFIG_LOG_LEVEL: DEBUG
INFO:config:DB_LOG_LEVEL: DEBUG
INFO:config:IMAGES_LOG_LEVEL: DEBUG
INFO:config:MAIN_LOG_LEVEL: DEBUG
INFO:config:MODELS_LOG_LEVEL: DEBUG
INFO:config:OLLAMA_LOG_LEVEL: DEBUG
INFO:config:OPENAI_LOG_LEVEL: DEBUG
INFO:config:RAG_LOG_LEVEL: DEBUG
INFO:config:WEBHOOK_LOG_LEVEL: DEBUG

Output:
INFO:     Started server process [17690]
INFO:     Waiting for application startup.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
user-join 5uKxxAqShH8N852ZAAAB {'auth': {'token': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImQ0NDIyYjM2LTU0MTMtNDQ4MC05ZTVmLTBmNGE0NjUzYmQyMiJ9.jVADet3qZppcSfBKSP1kRYSomyix0jCke-7EM-fQQRM'}}
user eddi(d4422b36-5413-4480-9e5f-0f4a4653bd22) connected with session ID 5uKxxAqShH8N852ZAAAB

generate_title
gemma2:latest
['5uKxxAqShH8N852ZAAAB']
['5uKxxAqShH8N852ZAAAB']

**Expected Behavior:**
debug logs

**Actual Behavior:**
no debug logs

**Comment by emmet7life** [comment]
me too. and i do not know why. i have try a lot of ways to solve it. but no result. is there anyone help us. thanks very much.

**Comment by tjbck** [comment]
Fixed on dev!
